app/components/developer_primary_action_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if business? %>
+  <%= link_to new_developer_message_path(@developer), rel: "nofollow", class: "shrink-0 mt-4 md:mt-0 inline-flex items-center px-4 py-2 border border-gray-300 shadow-sm text-base font-medium rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500" do %>
+    <%= inline_svg_tag "icons/solid/briefcase.svg", class: "-ml-1 mr-3 h-5 w-5" %>
+    <%= t("developers.show.hire") %>
+  <% end %>
+<% elsif owner? %>
```

Comment:
I think the priority needs to be reversed here. A user can be both a business _and_ developer and when they look at their own profile they should see Edit, not Hire.

Reply:
Actually, the check should be `if owner?` and `else`. We want to shot the Hire button even if the user isn't a business (yet).